backend/api/api_deployment_views.py

```diff
@@ -49,10 +49,9 @@ def post(
         file_objs = request.FILES.getlist(ApiExecution.FILES_FORM_DATA)
         serializer = ExecutionRequestSerializer(data=request.data)
         serializer.is_valid(raise_exception=True)
-        timeout = serializer.get_timeout(serializer.validated_data)
-        include_metadata = (
-            request.data.get(ApiExecution.INCLUDE_METADATA, "false").lower() == "true"
-        )
+        timeout = serializer.validated_data.get(ApiExecution.TIMEOUT_FORM_DATA)
+        include_metadata = serializer.validated_data.get(ApiExecution.INCLUDE_METADATA)
+        use_file_history = serializer.validated_data.get(ApiExecution.USE_CACHE)
```

Comment:
Why we use ApiExecution.USE_CACHE instaed of ApiExecution.USE_FILE_HISTORY. Would be better if we use same name to avoid confusions.

Reply:
Renamed the variable to `use_file_history` everywhere to avoid confusion